src/infinite-hits/__tests__/infinite-hits.spec.ts

```diff
@@ -46,11 +48,90 @@ describe('InfiniteHits', () => {
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadMore'
+    );
     button.click();
 
     expect(button.disabled).toBeTruthy();
     expect(showMore).not.toHaveBeenCalled();
     expect(fixture).toMatchSnapshot();
   });
+
+  it('should display `showPrevious()` button', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+
+    expect(button).not.toBeFalsy();
+  });
+
+  it('should display `showPrevious()` button with custom label', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template:
+        '<ais-infinite-hits [showPrevious]=true showPreviousLabel="Load previous"></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+
+    expect(button.innerHTML).toMatchInlineSnapshot(`" Load previous "`);
+  });
+
+  it('should call `showPrevious()` on button click', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+    button.click();
+
+    expect(button.disabled).toBeFalsy();
+    expect(showPrevious).toHaveBeenCalled();
+  });
+
+  it('should disable `showPrevious` button', () => {
+    const showPrevious = jest.fn();
+    const renderWithPrevious = createRenderer({
+      defaultState,
+      template: '<ais-infinite-hits [showPrevious]=true></ais-infinite-hits>',
+      TestedWidget: NgAisInfiniteHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+    const fixture = renderWithPrevious({ showPrevious, isFirstPage: true });
+
+    const button = fixture.debugElement.nativeElement.querySelector(
+      '.ais-InfiniteHits-loadPrevious'
+    );
+    button.click();
+
+    expect(button.className).toMatchInlineSnapshot(
+      `"ais-InfiniteHits-loadPrevious ais-InfiniteHits-loadPrevious--disabled"`
+    );
+    expect(button.disabled).toBeTruthy();
+    expect(showPrevious).not.toHaveBeenCalled();
+    expect(fixture).toMatchSnapshot();
+  });
```

Comment:
It would be nice to add a test featuring ng-template usage.
I see we're missing one for showMore already but `ng-template` usage is common enough that's it's worth adding a test for it.